### ISSUE TITLE
Build failure with GHC 8

### ISSUE THREAD

**Issue by snoyberg** [greydot/hdaemonize#3]
```
> /tmp/stackage-build8$ stack unpack hdaemonize-0.5.0.1
Unpacked hdaemonize-0.5.0.1 to /tmp/stackage-build8/hdaemonize-0.5.0.1/
> /tmp/stackage-build8/hdaemonize-0.5.0.1$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup configure --package-db=clear --package-db=global --package-db=/home/stackage/work/builds/nightly/pkgdb --libdir=/home/stackage/work/builds/nightly/lib --bindir=/home/stackage/work/builds/nightly/bin --datadir=/home/stackage/work/builds/nightly/share --libexecdir=/home/stackage/work/builds/nightly/libexec --sysconfdir=/home/stackage/work/builds/nightly/etc --docdir=/home/stackage/work/builds/nightly/doc/hdaemonize-0.5.0.1 --htmldir=/home/stackage/work/builds/nightly/doc/hdaemonize-0.5.0.1 --haddockdir=/home/stackage/work/builds/nightly/doc/hdaemonize-0.5.0.1 --flags=
Configuring hdaemonize-0.5.0.1...
> /tmp/stackage-build8/hdaemonize-0.5.0.1$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup build
Building hdaemonize-0.5.0.1...
Preprocessing library hdaemonize-0.5.0.1...
[1 of 1] Compiling System.Posix.Daemonize ( System/Posix/Daemonize.hs, dist/build/System/Posix/Daemonize.o )

System/Posix/Daemonize.hs:62:77: error:
    Module ‘System.Posix.Syslog’ does not export ‘syslog’
```

**Comment by greydot** [comment]
Thanks for reporting this. It looks like hsyslog API has changed. I'm gonna add an upper bound for now. New version will be released in the next few days.

**Comment by greydot** [comment]
http://hackage.haskell.org/package/hdaemonize-0.5.0.2

**Comment by greydot** [comment]
This is fixed in hdaemonize-0.5.2. Closing the issue for now.
